lib/system.g

```diff
@@ -109,6 +109,8 @@ BIND_GLOBAL( "GAPInfo", rec(
            help := [ "Run ProfileLineByLine(<filename>) on GAP start"] ),
       rec( long := "cover", default := "", arg := "<file>",
            help := [ "Run CoverageLineByLine(<filename>) on GAP start"] ),
+      rec( long := "ignoreconfig", default := false,
+           help := [ "disable/enable reading user configuration files"] ),
```

Comment:
So... this is not about this PR, but a general remark: if I do `gap --ignoreconfig --ignoreconfig` it does not ignore my config after all? I find that quite confusing ... I guess I find the whole "repeat an argument to toggle it" counter intuitive, but in this case it is particularly irritating. Can't we switch to offering `--ignoreconfig` and `--no-ignoreconfig` etc.?  (But again, i am not asking for this to be done in this PR).